src/lessons/07_css_html_part_2/css_html_part_2.slides.html

```diff
@@ -0,0 +1,547 @@
+                class="small"
+                >More on form element</a
+              >
+            </p>
+          </section>
+          <section>
+            <h2>&lt;input&gt;</h2>
+            <ul>
+              <li class="fragment">Is used to gather data from the user</li>
+              <li class="fragment">
+                How it works will be dependant of their attribute<span
+                  class="highlight"
+                  >type</span
+                >
+              </li>
+              <li class="fragment">
+                Every input should have an associated
+                <span class="highlight">label</span>
+              </li>
+            </ul>
+            <pre>
+              <code class="fragment formattedCode">
+                <script type="text/template">
+                  <label for="username">Username</label>
+                  <input type="text" name="username" id="username">
+                  <label for="quantity">Quantity</label>
+                  <input type="number" name="quantity" id="quantity">
+              </script>
+            </code>
+            </pre>
+            <p class="fragment">
+              <a
+                href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input"
+                class="small"
+                >More on input element</a
+              >
+            </p>
+          </section>
+          <section>
+            <h2>&lt;button&gt;</h2>
+            <ul>
+              <li class="fragment">Represents a clickable element</li>
+              <li class="fragment">
+                It generally indicates that an action can by done by pressing it
+              </li>
+              <li class="fragment">
+                If the button is not for submitting a form we need to set their
+                <span class="highlight">type</span> to
+                <span class="highlight">button</span>
+              </li>
+            </ul>
+            <pre>
+              <code class="fragment formattedCode">
+                <script type="text/template">
+                  <button>Submit!</button>
+                  <button type="button">It does submit</button>
+              </script>
+            </code>
+            </pre>
+            <p class="fragment">
+              <a
+                href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button"
+                class="small"
+                >More on button element</a
+              >
+            </p>
+          </section>
+          <section>
+            <h2>&lt;select&gt;</h2>
+            <ul>
+              <li class="fragment">It provides a menu of options</li>
+              <li class="fragment">
+                Each option has a value represented by the
+                <span class="highlight">&lt;select&gt;</span> tag
+              </li>
+            </ul>
+            <pre>
+              <code class="fragment formattedCode">
```

Comment:
Can you add the class `html` here? Highlight.js seems to mess up the syntax highlighting here
```suggestion
              <code class="fragment formattedCode html">
```